tools/querytee/response_comparator.go

```diff
@@ -179,7 +179,7 @@ func compareMatrix(expectedRaw, actualRaw json.RawMessage, opts SampleComparison
 
 		err := compareMatrixSamples(expectedMetric, actualMetric, opts)
 		if err != nil {
-			return err
+			return fmt.Errorf("%w\nFull expected result:\n%v\n\nFull actual result:\n%v", err, expectedMetric, actualMetric)
```

Comment:
(nit) why "full"? Just Expected result would be sufficient imo.

Reply:
I was trying to disambiguate it from the rest of the error message, which will refer to a single point.

Does 9875b9d look better to you?